contrib/linux-kernel/test/Makefile

```diff
@@ -5,21 +5,21 @@ SOURCES := $(wildcard ../lib/zstd/*.c)
 OBJECTS := $(patsubst %.c,%.o,$(SOURCES))
 
 ARFLAGS := rcs
-CXXFLAGS += -std=c++11
-CFLAGS += -g -O0
+CXXFLAGS += -std=c++11 -g -O3
+CFLAGS += -g -O3 -Wframe-larger-than=400
```

Comment:
Yeah it does, but the reviewer was concerned about stack usage, since the kernel stack is 16 KB at most. There were a few functions between 512 and 1024 bytes, so I fixed them up as well. After that, they were all below 400.